api/models/Notification.js

```diff
@@ -246,7 +246,8 @@ module.exports = bookshelf.Model.extend({
       'activity.reader',
       'activity.actor'
     ]})
-    .then(notifications => notifications.map(notification => notification.send()))
+    .then(notifications => Promise.map(notifications, notification => notification.send(), {concurrency: 10}))
```

Comment:
I mean concurrency between two or more different queued sendUnsent jobs

On Tue, May 31, 2016 at 12:42 PM Robbie Carlton <notifications@github.com>
wrote:

> In api/models/Notification.js
> <https://github.com/Hylozoic/hylo-node/pull/130#discussion_r65249191>:
>
> > @@ -246,7 +246,8 @@ module.exports = bookshelf.Model.extend({
> >        'activity.reader',
> >        'activity.actor'
> >      ]})
> > -    .then(notifications => notifications.map(notification => notification.send()))
> > +    .then(notifications => Promise.map(notifications, notification => notification.send(), {concurrency: 10}))
>
> @levity <https://github.com/levity> - like that? or concurrency somewhere
> else?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-node/pull/130/files/2d0f14e4e8cd87ff4b58706648864385b97418b1#r65249191>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAL0gyGhPJ85gUoYX2kQ1df0hGaTMQqtks5qHI8zgaJpZM4Iq3B->
> .
>